packages/api/src/hooks/useAccesiblePlatforms.ts

```diff
@@ -0,0 +1,22 @@
+import useLandingCompany from './useLandingCompany';
+
+/**
+ * A custom hook that provides flags to determine the accessibility status of cTrader and Dxtrade based on the current country of residence.
+ */
+const useAccesiblePlatforms = () => {
+    const { data: landing_company } = useLandingCompany();
+
+    /** check if ctrader jurisdiction is offered in the landing_company response  */
+    const is_ctrader_available = landing_company?.ctrader?.all?.standard === 'svg';
```

Comment:
Pls use the modified_accesible_platform format. Can check useLandingCompany for reference